params/config.go

```diff
@@ -504,6 +504,123 @@ func (c CliqueConfig) String() string {
 	return fmt.Sprintf("clique(period: %d, epoch: %d)", c.Period, c.Epoch)
 }
 
+// String implements the fmt.Stringer interface, returning a string representation
+// of ChainConfig that shows actual timestamp values instead of pointer addresses.
+func (c *ChainConfig) String() string {
+	result := fmt.Sprintf("ChainConfig{ChainID: %v", c.ChainID)
+
+	// Add block-based forks
+	if c.HomesteadBlock != nil {
+		result += fmt.Sprintf(", HomesteadBlock: %v", c.HomesteadBlock)
```

Comment:
you're not dereferencing the pointer and `%v` won't do it for you.

Reply:
It's `*big.Int`, so the go-fmt will print the number value for you